controller/area_blackbox_test.go

```diff
@@ -278,6 +278,12 @@ func (rest *TestAreaREST) TestShowChildrenArea() {
 			assertResponseHeaders(t, res)
 		})
 	})
+
+	rest.T().Run("Failure", func(t *testing.T) {
+		t.Run("Not Found", func(t *testing.T) {
+			test.ShowChildrenAreaNotFound(t, svc.Context, svc, ctrl, uuid.NewV4().String(), nil, nil)
```

Comment:
@jarifibrahim I highly recommend to use a golden file here for the response. At one point the messages in the golden files might play a role to UI.

Reply:
@kwk  we do not have golden files for any tests in area. I think we should create a new task and add all golden files for area test. What do you think?

Reply:
@jarifibrahim from my past experience those "let's create a new PR" after this one barely made it into reality. So please just add golden files here where you see fit. I know that it wasn't initially part of this PR but I think it is fine to do it all in one go.